TTreeValidation.cc

```diff
@@ -2318,9 +2360,9 @@ void TTreeValidation::fillConfigTree()
   fPhiFactor_ = Config::fPhiFactor;
   nEtaPart_   = Config::nEtaPart;
 
-  nlayers_per_seed_ = Config::nlayers_per_seed;
-  maxCand_ = Config::maxCandsPerSeed;
-  chi2Cut_ = Config::chi2Cut;
+  nlayers_per_seed_ = Config::ItrInfo[0].m_params.nlayers_per_seed;
+  maxCand_ = Config::ItrInfo[0].m_params.maxCandsPerSeed;
+  chi2Cut_ = Config::ItrInfo[0].m_params.chi2Cut;
```

Comment:
Should this get iteration id from somewhere, instead of hardcoding to 0?

Reply:
this is the configtree which is never used later in validation

I think we can fill it once per iteration in case we run multi-iteration setup